Makefile

```diff
@@ -64,15 +64,14 @@ integration-test: azurefile
 
 .PHONY: e2e-test
 e2e-test:
-	go test -v -timeout=0 ./test/e2e -e2e.bootstrap=${E2E_BOOTSTRAP} ${GINKGO_FLAGS}
+	go test -v -timeout=0 ./test/e2e ${GINKGO_FLAGS}
 
 .PHONY: e2e-bootstrap
 e2e-bootstrap: install-helm
 	docker pull $(IMAGE_TAG) || make azurefile-container push
 ifdef TEST_WINDOWS
 	helm install azurefile-csi-driver charts/latest/azurefile-csi-driver --namespace kube-system --wait --timeout=15m -v=5 --debug \
-		--set image.azurefile.repository=$(REGISTRY)/$(IMAGE_NAME) \
-		--set image.azurefile.tag=$(IMAGE_VERSION) \
+		${E2E_HELM_OPTIONS} \

```

Comment:
I think you also need to set `${E2E_HELM_OPTIONS}` in line 80, line 74 is for windows e2e test
@chewong